modules/gdscript/tests/scripts/analyzer/errors/function_used_as_property.gd

```diff
@@ -1,6 +1,5 @@
 func function():
 	pass
 
-
 func test():
```

Comment:
This change goes contrary to the official GDScript style guide.

Reply:
There's a lot of other line removals that break the style guide as well.

Reply:
I'll remove any changes about line removals.